packages/ckeditor5-upload/src/filerepository.ts

```diff
@@ -148,17 +148,10 @@ export default class FileRepository extends Plugin {
 			 * This warning shows up when {@link module:upload/filerepository~FileRepository} is being used
 			 * without {@link module:upload/filerepository~FileRepository#createUploadAdapter defining an upload adapter}.
 			 *
-			 * **If you see this warning when using one of the {@glink getting-started/legacy/installation-methods/predefined-builds
-			 * CKEditor 5 Builds}**
-			 * it means that you did not configure any of the upload adapters available by default in those builds.
-			 *
-			 * See the {@glink features/images/image-upload/image-upload comprehensive "Image upload overview"} to learn which upload
-			 * adapters are available in the builds and how to configure them.
-			 *
-			 * **If you see this warning when using a custom build** there is a chance that you enabled
-			 * a feature like {@link module:image/imageupload~ImageUpload},
-			 * or {@link module:image/imageupload/imageuploadui~ImageUploadUI} but you did not enable any upload adapter.
-			 * You can choose one of the existing upload adapters listed in the
+			 * If you see this warning there is a chance that you enabled a feature like
+			 * {@link module:image/imageupload~ImageUpload}, or {@link module:image/imageupload/imageuploadui~ImageUploadUI}
+			 * but you did not enable any upload adapter. You can choose one of the existing upload
```

Comment:
Why did we delete the predefined mention? I would go the same way we went with previous messages. It's still top 2 error which could fuel migration :D

Reply:
OK